custom/icds_reports/management/commands/pull_Data_school.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+
+from django.core.management.base import BaseCommand
+from corehq.form_processor.interfaces.dbaccessors import CaseAccessors
+from django.db import connections, transaction
+from dateutil.relativedelta import relativedelta
+from corehq.sql_db.connections import get_icds_ucr_citus_db_alias
+
+
+def _run_custom_sql_script(command):
+    db_alias = get_icds_ucr_citus_db_alias()
```

Comment:
you can also just fetch it once globally.

Reply:
sure